sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -115,6 +115,37 @@ extern "C" {
 #define GNRC_SIXLOWPAN_ND_AR_LTIME          (15U)
 #endif
 
+/**
+ * @brief   Size of the virtual reassembly buffer
+ *
+ * @see     https://tools.ietf.org/html/draft-ietf-lwig-6lowpan-virtual-reassembly-01
+ *
+ * @note    Only applicable with
+ *          [gnrc_sixlowpan_frag_vrb](@ref net_gnrc_sixlowpan_frag_vrb) module,
+ *          but has also a direct influence on the number of available
+ *          gnrc_sixlowpan_rbuf_int_t entries (even when
+ *          `gnrc_sixlowpan_frag_vrb` is not compiled in).
+ */
+#ifndef GNRC_SIXLOWPAN_FRAG_VRB_SIZE
+#if defined(MODULE_GNRC_SIXLOWPAN_FRAG_VRB) || defined(DOXYGEN)
+#define GNRC_SIXLOWPAN_FRAG_VRB_SIZE        (16U)
+#else   /* defined(MODULE_GNRC_SIXLOWPAN_FRAG_VRB) || defined(DOXYGEN) */
+#define GNRC_SIXLOWPAN_FRAG_VRB_SIZE        (0U)
+#endif  /* defined(MODULE_GNRC_SIXLOWPAN_FRAG_VRB) || defined(DOXYGEN) */
+#endif  /* GNRC_SIXLOWPAN_FRAG_VRB_SIZE */
```

Comment:
I'm doing this so complicated, to allow for easy fragment interval multiplicity calculation in #11068 (and coming fragment forwarding solutions):

https://github.com/RIOT-OS/RIOT/blob/7d542517c89174f06ccfdf55d482c4438b72b290/sys/net/gnrc/network_layer/sixlowpan/frag/rbuf.c#L50-L51

If if `gnrc_sixlowpan_frag_vrb` is not used `GNRC_SIXLOWPAN_FRAG_VRB_SIZE` is 0 so I just use the fragment intervals required for the real reassembly buffer.